.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  - push
+
+name: Run PHPStan checks
+
+jobs:
+  mutation:
+    name: PHPStan ${{ matrix.php }}-${{ matrix.os }}
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+
+        php:
+          - "8.1"

```

Comment:
```suggestion
```
This list must match the list of supported PHP versions from [composer.json](https://github.com/dotkernel/dot-geoip/blob/3.0/composer.json#L25)